recipe-client-addon/bootstrap.js

```diff
@@ -90,9 +92,13 @@ this.shutdown = function(data, reason) {
     "lib/SandboxManager.jsm",
     "lib/Storage.jsm",
   ];
-  for (const module in modules) {
+  for (const module of modules) {
+    log.debug(`Unloading ${module}`);
     Cu.unload(`resource://shield-recipe-client/${module}`);
   }
+
+  // Don't forget the logger!
+  log = null;
```

Comment:
I don't think we need this. It will fall out of scope and be removed at the end of the function anyway.